set.mm

```diff
@@ -45178,21 +45178,8 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
        to the standard Kuratowski definition; otherwise, it is defined to be
        the empty set.  Avoid directly depending on this detail so that theorems
        will not depend on the Kuratowski construction.  (Contributed by Mario
-       Carneiro, 26-Apr-2015.)  Avoid ~ ax-10 , ~ ax-11 , ~ ax-12 .  (Revised
-       by SN, 1-Aug-2024.)  (Avoid depending on this detail.) $)
+       Carneiro, 26-Apr-2015.)  (Avoid depending on this detail.) $)
     dfopif $p |- <. A , B >. =
-      if ( ( A e. _V /\ B e. _V ) , { { A } , { A , B } } , (/) ) $=
-      ( vx vy cop cvv wcel wa csn cpr c0 cif cv w3a wn wo noel intnan 3bitr4i
-      wb biorf ax-mp df-3an orcom weq eleq1 3anbi3d df-op elab2g elv elif eqriv
-      ) CABEZAFGZBFGZHZAIABJJZKLZUNUOCMZUQGZNZUPUTHZUPOZUSKGZHZPZUSUMGZUSURGVBV
-      EVBPZVAVFVEOVBVHTVDVCUSQRVEVBUAUBUNUOUTUCVBVEUDSVGVATCUNUODMZUQGZNVADUSUM
-      FDCUEVJUTUNUOVIUSUQUFUGDABUHUIUJUPUSUQKUKSUL $.
-    $( $j usage 'dfopif' avoids 'ax-10' 'ax-11' 'ax-12'; $)
```

Comment:
We want to keep the `$( $j usage 'dfopif' avoids 'ax-10' 'ax-11' 'ax-12'` even if we are restoring the old proof, right?